src/ansys/dpf/core/operators/result/accu_eqv_creep_strain.py

```diff
@@ -65,12 +65,6 @@ class accu_eqv_creep_strain(Operator):
     requested_location : str, optional
         Requested location nodal, elemental or
         elementalnodal
-    read_cyclic : int, optional
```

Comment:
@rafacanton @ricardopeixotocoelho is this normal? Why do we lose a pin?

Reply:
Yes, @ricardopeixotocoelho did the digging job to only support cyclic for the variables where cyclic is allowed. So this was a previous bug, basically